nimsuggest/tests/twithin_macro_prefix.nim

```diff
@@ -165,7 +168,7 @@ macro class*(head, body: untyped): untyped =
 class Animal of RootObj:
   var name: string
   var age: int
-  method vocalize: string {.base.} = "..." # use `base` pragma to annonate base methods
+  method vocalize: string {.base.} = return "..." # use `base` pragma to annonate base methods
```

Comment:
why this change? didn't work without it?

Reply:
IIRC yes that's why I had to make it, but I'll double check.

Reply:
Turns out it's not necessary will remove in future commit.